src/AccessControl/userfolder.py

```diff
@@ -111,7 +114,7 @@ def _doDelUsers(self, names):
     def identify(self, auth):
         if auth and auth.lower().startswith('basic '):
             try:
-                name, password = decodestring(auth.split(' ')[-1].encode()).decode() \
+                name, password = decodebytes(auth.split(' ')[-1].encode()).decode() \
```

Comment:
We just compared `auth` against native str above:  are you confident it is really bytes?

Reply:
According to the documentation (https://docs.python.org/3/library/base64.html#base64.decodestring) `base64.decodestring` is a deprecated alias of `decodebytes`. I think at least my change does not make the code worse.

Reply:
OK, but if the Python3 stdlib is expecting bytes, we shouldn't be passing in `auth` as text, nor comparing / splitting it with `str` constants.

Reply:
I've opened #56 to track this issue.

Reply:
@tseaver Thank you.

Reply:
Fixed in https://github.com/zopefoundation/AccessControl/commit/ad391da2cceb9d286d824c457479112e8743f74a